semantic_model_generator/snowflake_utils/env_vars.py

```diff
@@ -2,7 +2,7 @@
 
 from dotenv import load_dotenv
 
-load_dotenv()
+load_dotenv(override=True)
```

Comment:
intentional change, I realized `.env` was not overriding the values from zshrc unless this was set.